src/main/java/com/rewedigital/composer/routing/RouteMatch.java

```diff
@@ -11,14 +15,18 @@
     private final Map<String, Object> parsedPathArguments;
 
     public RouteMatch(final Match backend, final Map<String, String> parsedPathArguments) {
-        this.backend = backend;
+        this.backend = requireNonNull(backend);
         this.parsedPathArguments = Collections.<String, Object>unmodifiableMap(parsedPathArguments);
     }
 
     public String backend() {
         return backend.backend();
     }
 
+    public Optional<Duration> ttl() {
```

Comment:
Might be me not understanding the coverage report, but it looks like there is not test covering this new method?

Reply:
And actually, I don't find the code using the route timeout?

Reply:
Done